### DISCUSSION TITLE
Milvus Insight cannot load a vector without commas

### DISCUSSION THREAD

**Discussion by xqxq-2020** [milvus-io/milvus#16339] [Q&A and General discussion]
I can use this vector directly when I call collection.search(), but not in Milvus Insight. Is there any way to convert this vector directly into a method that can be used directly in Milvus Insight

![image](https://user-images.githubusercontent.com/62696048/161191560-43e5c242-cd3d-48e4-9db1-715c99358312.png)

![image](https://user-images.githubusercontent.com/62696048/161191585-0e5d313b-5316-407a-b548-37631ec06646.png)

![fcb564ee85a61e408762715ee79f274](https://user-images.githubusercontent.com/62696048/161191594-42b4e5d3-548c-4920-b26c-9703d9745672.png)

**Comment by nameczz** [comment]
If you are using Milvus 2.x, we recommend to use Attu. Insight is no longer updated。
Vector need to be separated by commas in Insight.
In my opinion, you need spilt with one space ,then join with commas and copy to Insight
